src/bitwise/core.cljs

```diff
@@ -87,6 +89,7 @@
                      {})}
        [:div (:pid process)]
        [:div (:name program)]
+       ; TODO: Add an "output column here"
```

Comment:
This doesn't seem to be part of this PR

Reply:
; TODO: Add a TODO to remove this "TODO" in a future PR.